src/Compilers/CSharp/Portable/Symbols/Source/SourceNamespaceSymbol.cs

```diff
@@ -500,28 +500,31 @@ public override bool IsDefinedInSourceTree(SyntaxTree tree, TextSpan? definedWit
             return false;
         }
 
-        private readonly struct NameToSymbolMapBuilder
+        private readonly ref struct NameToSymbolMapBuilder
```

Comment:
>ref [](http://example.com/codeflow?start=25&length=3)

It feels like conversion to ref struct is not warranted

Reply:
Also, given that this type is used only in one place and is used locally, we can get rid of the type completely

Reply:
done.